### ISSUE TITLE
[Adaptive Card iOS - ProductVideo.json]: Voice Over focus is not in sequential order on the video player.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#168]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Scenarios' control and activate it.
5. Navigate to 'VideoPlayer.json' under 'Scenarios' list item and activate it.
6. Card will be appear.
7. Perform right swipe and navigate to the video player and double tap to activate it.
8. Now, navigate the control present under player and observe the issue.

**Actual Result:**
Focus order is not in sequential order under the video player. As while navigating using swipe gesture, it is going on the go backward 10 seconds button after that enter full screen after that 0 colon 05 collapsed and so on.

**Expected Result:**
Voice over focus order should be in sequential order on the video player. It should be left to right manner.

**User Impact:**
It will impact the screen reader user as they will get confused if the focus order is not in sequential order on the video player.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/55b01658-48ce-4567-a002-77cdca87aa94

### Card JSON

```JSON
NA
```
